snapshotter/internal/http_address_resolver.go

```diff
@@ -56,7 +51,7 @@ func init() {
 //
 // Response:
 // {
-//     "network": "tcp",
+//     "network": "unix",
 //     "address": "192.168.0.1:80"
```

Comment:
nit: should this address be a path?

Reply:
Good call out. I was working to spin up a VM to get an example Vsock path and got sidetracked with the kernel upgrade issue. I'll work on this again and get an update here.